src/utilities.js

```diff
@@ -1,36 +1,14 @@
 import R from 'ramda';
 import { NpmConfig, httpsGetPromise } from './helpers';
 
-const addVersionProp = (val, key, obj) => obj[key] = { 'version': removeCaret(val) };
 const removeCaret = R.replace(/\^/, '');
+const addVersionProp = (v, k, o) => o[k] = { 'version': removeCaret(v) };
 const isNotPrivate = R.compose(R.not, R.prop('private'));
-const filterPrivate = R.filter(isNotPrivate);
-
-const deeplyMerge = (obj1, obj2) => {
-  return R.keys(obj1).reduce((result, key) => {
-    result[key] = R.merge(obj1[key], obj2[key]);
-    return result;
-  }, {});
-};
-
-export const pickDownloads = (obj) => {
-  return R.keys(obj).reduce((result, key) => {
-    result[key] = R.pick(['downloads'], obj[key]);
-    return result;
-  }, {});
-};
-
-export const addNode = (obj) => {
-  obj.node = {
-    name        : 'Node.js',
-    url         : 'https://nodejs.org',
-    version     : process.versions.node,
-    description : 'A JavaScript runtime ✨🐢🚀✨',
-    downloads   : 10000000 // A fake number since Node isn't downloaded on npm
-  };
-  return obj;
-};
+const deeplyMerge = (obj1, obj2) => R.mapObjIndexed((v, k, o) => R.merge(obj1[k], obj2[k]), obj1);
```

Comment:
I wouldn't classify it as safe, but in this context it's safe enough. In the long run, especially in the event it were to go into an environment with broader application, I'd like it to be more clever about how it reconciles collisions. In this app, given the nature of how truly deeply we're merging (one level) and the relatively fixed nature of our dependencies it appears sane.